### PR TITLE
rpmbuild: add page

### PR DESCRIPTION
**Pull request by hellojukay**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/rpmbuild.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# rpmbuild
+
+> RPM Package Build tool.
+
+- Build binary and source packages:
+
+`rpmbuild -ba {{spec file}}`
```
Comment (owenvoke) [inline pages/linux/rpmbuild.md:7]:
```suggestion
`rpmbuild -ba {{path/to/spec_file}}`
```

File: pages/linux/rpmbuild.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+- Build a binary package without source package:
+
+`rpmbuild -bb {{spec file}}`
```
Comment (owenvoke) [inline pages/linux/rpmbuild.md:11]:
```suggestion
`rpmbuild -bb {{path/to/spec_file}}`
```

File: pages/linux/rpmbuild.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+- Pass variables when building package:
+
+`rpmbuild -bb {{spec file}}  --define "{{variable1}} {{value1}}" --define "{{variable2 value2}}"`
```
Comment (owenvoke) [inline pages/linux/rpmbuild.md:15]:
```suggestion
- Specify additional variables when building a package:

`rpmbuild -bb {{path/to/spec_file}}  --define "{{variable1}} {{value1}}" --define "{{variable2 value2}}"`
```

**Review by owenvoke — CHANGES_REQUESTED** [review]
Hi @hellojukay, thanks for the new page. Just a few minor changes to the syntax, but otherwise looks good.

Also, do you know if there's a good official link that could be added to the page? 🤔

**Review by owenvoke — APPROVED** [review]

File: pages/linux/rpmbuild.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# rpmbuild
+
+> RPM Package Build tool.
+> More information: <https://docs.fedoraproject.org/en-US/quick-docs/creating-rpm-packages/index.html>.
```
Comment (navarroaxel) [inline pages/linux/rpmbuild.md:4]:
```suggestion
> More information: <https://docs.fedoraproject.org/en-US/quick-docs/creating-rpm-packages/>.
```

File: pages/linux/rpmbuild.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+`rpmbuild -ba {{path/to/spec_file}}`
+
+- Build a binary package without source package:
+
+`rpmbuild -bb {{path/to/spec_file}}`
+
+- Specify additional variables when building a package:
+
+`rpmbuild -bb {{path/to/spec_file}}  --define "{{variable1}} {{value1}}" --define "{{variable2 value2}}"`
```
Comment (navarroaxel) [inline pages/linux/rpmbuild.md:16]: There is an extra space and the `{{}}` are not consistent.
```suggestion
`rpmbuild -bb {{path/to/spec_file}} --define "{{variable1}} {{value1}}" --define "{{variable2}} {{value2}}"`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @hellojukay, just 2 comments.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**